dev/test_data/TilesetWithDiscreteLOD/README.md

```diff
@@ -0,0 +1,16 @@
+# Tileset with discrete LODs
```

Comment:
Could this folder be renamed from `test_data` to `sample_data`? That way, it's apparent that this data isn't used in tests at all.